plugins/reporters/static-html/src/main/kotlin/StaticHtmlReporter.kt

```diff
@@ -730,3 +706,20 @@ private val SCOPE_EXCLUDE_LIST_COMPARATOR = compareBy<Map.Entry<String, List<Sco
 private val PathExclude.description: String get() = joinNonBlank(reason.toString(), comment)
 
 private val ScopeExclude.description: String get() = joinNonBlank(reason.toString(), comment)
+
+private fun IssueTable.title(): String {
+    val type = when (type) {
+        IssueTable.Type.ANALYZER -> "Analyzer"
+        IssueTable.Type.SCANNER -> "Scanner"
+    }
+
+    return "$type Issue Summary ($errorCount errors, $warningCount warnings, $hintCount hints to resolve)"
+}
+
+private fun IssueTable.id(): String =
+    when (type) {
+        IssueTable.Type.ANALYZER -> "analyzer-issue-summary"
+        IssueTable.Type.SCANNER -> "scanner-issue-summary"
+    }
```

Comment:
Can be simplified to `"${type.name.lowercase()}-issue-summary"`.

Reply:
Did you forget to change this?